locales/da.js

```diff
@@ -90,7 +104,9 @@ shipping_address:
 payment_informations:
 "Betalingsinformation",
 payment_informations_bill_me_later:
-"Betal senere",
+"Betaling ikke nødvendig",
+payment_informations_no_payment_required:
+"",

```

Comment:
Is there an error here?
From google translate, `Betal senere` seemed correct for Pay Later and the new translation seems to fit `payment_informations_no_payment_required`

```suggestion
"Betaling ikke nødvendig",
```

Reply:
@JeanSebTr It does seem like there's an error. It should be as below :) does that help?

> payment_informations_bill_me_later:
> "Betal senere",
> payment_informations_no_payment_required:
> "Betaling ikke nødvendig",

Reply:
Great! Just wanted to be sure!